### PR TITLE
Automagically use curl instead of wget if that's what's available

### PR DESCRIPTION
**Pull request by eherot**
This should make things run a little more smoothly on Macs.

### CODE REVIEW

File: addons/cluster-autoscaler/cluster-autoscaler.sh [hunk 1]
```diff
@@ -91,7 +91,16 @@ aws iam attach-role-policy --policy-arn $ASG_POLICY_ARN --role-name $IAM_ROLE
 printf " ✅ \n"
 
 addon=cluster-autoscaler.yml
-wget -O ${addon} https://raw.githubusercontent.com/kubernetes/kops/master/addons/cluster-autoscaler/v1.8.0.yaml
+manifest_url=https://raw.githubusercontent.com/kubernetes/kops/master/addons/cluster-autoscaler/v1.8.0.yaml
+
+if which -s wget; then
```
Comment (justinsb) [inline addons/cluster-autoscaler/cluster-autoscaler.sh]: Unfortunately I don't think `-s` is supported on linux. 

You could use this trick: https://github.com/kubernetes/kubernetes/blob/6696ae39ee7df5323e843d93c6146e51d297a299/cluster/get-kube-local.sh#L132

**Comment by justinsb** [comment]
I don't think we can use `-s` I'm afraid, but otherwise looks good

**Comment by justinsb** [comment]
(This isn't bundled into the release artifact, so I'm putting this into the 1.12 milestone because it doesn't need to impact 1.11 release timelines)

**Comment by eherot** [comment]
What's the issue with `-s`? I would think it would be pretty annoying to have the data transfer stats muddying the output...

**Comment by justinsb** [comment]
@eherot does my comment about which not supporting `-s` make sense?  I think you can just use the `$(...)` trick I linked instead

**Comment by eherot** [comment]
@justinsb Yes! Just got distracted and haven't implemented your suggestion yet but I do plan to do so.

**Comment by justinsb** [comment]
Thanks @eherot 

/approve
/lgtm

**Pull request merged**